### DISCUSSION TITLE
List of ReProver proofs on benchmarks?

### DISCUSSION THREAD

**Discussion by darabos** [lean-dojo/ReProver#32] [Q&A] [answered]
Hi,
I see a list of 33 new MiniF2F proofs from ReProver in https://github.com/facebookresearch/miniF2F/pull/13. I was wondering if the whole set of ReProver-generated proofs was available somewhere? It would be useful for comparison with baseline proofs (on LeanDojo Benchmark) and with proofs from other automated provers.
Thanks!

**Comment by yangky11** [comment, marked as answer]
Hi,

I just re-run ReProver (Lean 3) on miniF2F's test set and included the evaluation log [here](https://github.com/lean-dojo/ReProver/blob/legacy/logs/eval_reprover_minif2f_test_8gpus.out). It proves 27.8% test theorems (slightly better than our previous result). You can parse the log to see which theorems are successfully proved.

**Reply by darabos** [reply]
Perfect, thanks so much!

(I've extracted it into a [dict](https://gist.github.com/darabos/4e8e422b29a258662bb301999e97bf09).)
